src/user/controller/by-href.ts

```diff
@@ -13,6 +13,7 @@ class UserByHrefController extends Controller {
 
     let hasControl = false;
     let hasPassword = false;
+    let hasPrivilege = false;
```

Comment:
Would suggest:

```typescript
const isAdmin = await privilegeService.hasPrivilege(ctx, 'admin');
```

Now you have a single variable that can be re-used later in this function too.